tests/e2e/ios-swift/README.md

```diff
@@ -0,0 +1,31 @@
+# RudderTyper - iOS (Swift)
+
+This is a sample iOS (Swift) app designed to demonstrate how to use the RudderStack iOS SDK to track the RudderTyper generated events.
+
+## Prerequisites
+
+- Download, install and setup [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
+
+## Getting Started
```

Comment:
I think we must add the steps to perform `pod install` and then open the `RudderTyperSwiftExample.xcworkspace`. Please update the step to the documentation.